Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -197,6 +197,9 @@ public enum SentrySDK {
         event.message = "This is a crash with stowed exceptions. The events are grouped by the stack trace of the latest stowed exception.\n" +
                         "You can find the crash stack of the other stowed exceptions and of the outer crash by scrolling down."
         let eventSerialized = event.serialized()
+        // The list of loaded modules is cached and might not contain the modules that were loaded after initializing Sentry (e.g. the WinApp runtimes).
+        // Clearing the module cache will force Sentry to re-fetch the list of loaded modules.
+        sentry_clear_modulecache()
```

Comment:
Hmmm, interesting. I wonder if it is safe to do this while handling the exception, since memory can be in invalid state depending on the exception. Not sure if sentry_clear_modulecache is safe to call.

Reply:
This is the doc for `sentry_clear_modulecache`:
```
/**
 * Clears the internal module cache.
 *
 * For performance reasons, sentry will cache the list of loaded libraries when
 * capturing events. This cache can get out-of-date when loading or unloading
 * libraries at runtime. It is therefore recommended to call
 * `sentry_clear_modulecache` when doing so, to make sure that the next call to
 * `sentry_capture_event` will have an up-to-date module list.
 */
 ```
 
Rebuilding the cache is pretty safe afaik, it's just an optimization feature. I'm also not too concerned about the app's state during a stowed exception report, there could be some memory corruption issue but if we get to this crash handler then there's no strong reason to believe that doing a module enumeration will fail. Stowed exceptions are usually triggered by failed WinUI invariants, not corrupt app state.
 
The side effect of *not* rebuilding the cache means that we won't get any symbol for stack frames in the Win App runtimes during stowed exception crashes.

Reply:
Ok!